src/components/Header/ProfileBtn.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import ProfileImg from '../../asset/images/profile.jpg';
+
+const ProfileBtnWrap = styled.button`
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 105px;
+    height: 32px;
+    padding: 2px;
+    border-radius: 20px;
+    position: absolute;
+    top: 50%;
+    right: 17px;
+    transform: translateY(-50%);
+    background-color: #080810;
+    cursor: pointer;
+
+    &:hover {
+        background-color: #393939;
+    }
+`;
+
+const ProfileBtnImg = styled.div`
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    background: url(${ProfileImg}) top left no-repeat;
+    background-size: cover;
+`;
+
+const ProfileName = styled.div`
+    color: #fff;
+    font-size: 0.75rem;
+    font-weight: 900;
+`;
+
+const ProfileArrow = styled.div`
+    width: 20px;
+    height: 10px;
+    margin-top: 8px;
+    position: relative;
+    overflow: hidden;
+    &::after {
+        content: '';
+        display: block;
+        width: 7px;
+        height: 7px;
+        position: absolute;
+        top: -30%;
+        left: 50%;
+        background-color: #fff;
+        transform: translateX(-50%) rotate(45deg);
+    }
+`;
+
+interface IProps {
+    onClick: () => void;
```

Comment:
```suggestion
    onClick?: () => void;
```
이런식으로 꼭필요한 요소아니면 옵셔널하게 받으면 나중에(물론 이 프로젝트에는 해당안되는거 같음) 재사용할때나 스토리북 사용할때 더 좋을 것 같아요

Reply:
보통 대비해서 옵셔널하게 타입을 주는경우가 스태틱하게 주는경우보다 많나요??😮

Reply:
설계상의 문제인데 혹시 데이터를 받아왔을때 문제가 있는경우를 대비해서 옵셔널하게 받기도합니다.
아 물론 옵셔널하게 안받고 이런식으로도 가능합니다
`value:string | null`